README.md

```diff
@@ -61,15 +71,18 @@ CONNECTOR_LEDGERS={
     "currency": "EUR",
     "plugin": "ilp-plugin-payment-channel-framework",
     "options": {
-      "prefix": "g.eur.mytrustline.",
-      "token": "shared_secret", // shared secret between server and client
-      "rpcUri": "https://wallet1.example/api/peers/rpc",
+      "server": "btp+wss://wallet1.example:1234/example_path",
+      "token": "shared_secret"
     },
     "store": false
   }
 }
 ```
 
+> **Note:** You can provide both the `server` and `listener` properties in which
+> case the BTP peers will automatically elect one of them to be the server and
+> one of them to be the client.
```

Comment:
I don't think that's useful. It means you need to exchange server URLs in both directions, even if only one gets used?

And what if you have >= 1 peer? Will you need to put each peer on a different port?

Reply:
See also https://github.com/interledgerjs/ilp-plugin-payment-channel-framework/issues/29